HTML/4. text.md

```diff
@@ -0,0 +1,93 @@
+# Text Styling
+
+You can style text in HTML in multiple different ways, you can make it bold, italicized, underlined, striketrough, superscript and subscript.
+___
+## Bold 
+You can do bold in two ways, the `<b>` tag or the `<strong>` tag, in terms of HTML semantics you should use `<strong>`
+
+Here's how it would look rendered on a webpage
+
+Code:
+
+```html
+<p>Hello <strong>how</strong> are you?</p>
+```
+
+Output:
+
+
+![Bold Text Picture](images/bold.png)
+
+**Look [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#inline_text_semantics) for more information about HTML Semantics**
+___
+## Italics
+You can do Italics through two tags, the `<i>` tag or the `<em>` tag, and `<em>` is semantically correct.
```

Comment:
Again talk about how em is used to emphasize something while i is just used to make something italic